### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by danielepolencic** [learnk8s/learnk8s.io#47]
Kubernetes has a weekly newsletter: [KubeWeekly](http://kube.news/). This is a dump of every link mentioning the word Kubernetes. I find it very hard to follow and not very informative.

There's a second and very young newsletter called [Kubelist](https://kubelist.com/). The project just started, and I can count only three issues. It's curated ✅and human-friendly. But I think it lacks focus. It's about "all things Kubernetes", same as Kubeweekly.

There's a third newsletter [KLWD](http://lwkd.info/) - Last Week in Kubernetes Development. The newsletter is focussed primarily on maintainers and contributors.

The k8s community is not as mature as other communities. JS and Ruby have a long history when it comes to newsletters, and they have a lot of insights to offer when it comes to helping the community broadcasting news.

This is an opportunity for us to bring some of the learnings and the ❤️from other communities to k8s.

I spent some time thinking about what I'd like to read every week if I were:

1. a k8s newbie and
1. someone who wants to develop applications on k8s

I also did a lot of research into other newsletters. I targeted three large communities I'm familiar with:

- Javascript
- Ruby
- Designers

You can explore newsletters from more communities from this [awesome newsletter collection](https://github.com/vredniy/awesome-newsletters).

My findings in a nutshell:

- A limited set of links. No one has time to read a wall of text
- Curated content. I think this is probably the most significant learning for me. While the newsletter is already k8s focussed, it should be focussed on a particular aspect of k8s. [KLWD](http://lwkd.info/) is a good example: just news on development.
- No Spam. Goes without saying.
- No one wants to subscribe if they can't see the quality of the newsletter. But I couldn't find evidence of what _quality_ means.
- Author or curator play a significant role when it comes to trusting the content or quality of the newsletter
- Newsletters are much more than just a list of links. There's an opportunity to share events, funny facts, etc.

So I came up with a newsletter idea in line with what learnk8s does best: helping others to get up to speed with k8s.

![image](https://user-images.githubusercontent.com/1130499/37098368-a17ccd70-2215-11e8-9a9f-801447fd0e8d.png)

And this is the full newsletter

![image](https://user-images.githubusercontent.com/1130499/37098626-41f74e9c-2216-11e8-9a3b-a48ac09fe89b.png)

You may be asking... what about the content?

I collect daily news from r/kubernetes and twitter about Kubernetes. All the bookmarks I save are stored in Diigo.
I reckon we could get a digest of the activity of the Twitter account and the bookmarks I saved during the week to curate a newsletter.
Of course, the retweets, favs, bookmarks have to be learning k8s focussed.

Let me know what you think! @salmaniqbal @denhamparry @chrisns @valentin2105 @keithmifsud

**Comment by denhamparry** [comment]
These are what I look for in Newsletters / Publications:

- Latest news (< 5 mins / break reads)
- Ideas, concepts and documentation (< 15 mins / lunchtime reads)
- Tutorials (< 1 hour / personal development)

With that point, an example of a tutorial would be "Getting started with _insert latest buzzwork_ on Google Kubernetes Engine".  The content would need to be created / curatetd by learnk8s but feel that this can be reused else where to further promote learnk8s.  Maybe this should be a separate issue?

**Comment by keithmifsud** [comment]
@danielepolencic Sorry I meant:

When I receive a newsletter, I either have time to skim through it there and then or if I don't but see an interesting heading, I'll __star__ _(not start)_ it for later.

**Comment by keithmifsud** [comment]
@danielepolencic I think the reading time and reader's interest can be a bit of a hit and miss. An article might be long, and it can also be exciting to the reader, but if the reader is too busy upon opening the email or is just trying to clean up their inbox, then they may still not be able to read it.

Needless to say that longer articles should have an even more compelling headline and excerpt.

**Comment by almoretti** [comment]
Alright, good conversation on content all valid assumptions but it will all depend on the audience that actually subscribes so we will learn on the go. 

@danielepolencic To get this off the ground and actually get some activation plan in place to start to send out some e-mail we have 3 streams we need to work on :

1) Email marketing service - do we have one? A free and easy pick could be Mailchimp or if you want to go open source there is mautic. Probably I'll go MailChimp as it's mainstream and integrated with a lot of other techs out of the box 
2) E-mail List generation -  I'm sure right now all those e-mails will sit in silos. (i.e on a paper form, or on sheet o someone computer).  Where do we collect all the e-mail, can we list out all the active and potential sources of e-mail we could get online and offline? (i.e pop-up on the blog, website widget, form during a 2h workshop)
3)Assets - We will need specific email templates. The first one is for the welcome journey, a Simple template that will confirm the user that he subscribes to the newsletter and further marketing communications. The second one is the actual newsletter. This should be built in a modular way, where we agree on the modules that we want to use, the frequency etc.

**Comment by keithmifsud** [comment]
@almoretti I can see that we're already using Mailchimp for the subscription. You can see the form and subscribe here (above footer) https://learnk8s.io/blog/smaller-docker-images

**Comment by keithmifsud** [comment]
@danielepolencic The subscribe button says "Join 1500+ others). If we have already have over 1500, then note that the free plan from Mailchimp has a 2000 subscriber limit. Let me know if you need me to do some work on this.

**Comment by almoretti** [comment]
Hi, as far as I know we are not using MailChimp

I set up the service with sendblue

**Comment by keithmifsud** [comment]
Superseded by #89
